recipe/meta.yaml

```diff
@@ -1,25 +1,30 @@
 {% set name = "shap" %}
-{% set version = "0.44.0" %}
+{% set version = "0.44.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 8f5f0401df83c8b2b566f41872f1bd8bc05bba43aab9e82bfb2047e9b2a0471c
+  sha256: a21d5a622e12e7c3b4a58d6e93b70133b7e09b6342b19746071a0dc2d190b432
   patches:
     - 0001-Build-for-all-CUDA-11-arches.patch  # [cuda_compiler_version == "11.8"]
     - 0001-Build-for-all-CUDA-12-arches.patch  # [(cuda_compiler_version or "").startswith("12")]
     # Taken and adapted from https://github.com/shap/shap/pull/3311
     - 0002-remove-old-kepler-architecture-for-cuda-build.patch  # [(cuda_compiler_version or "").startswith("12")]
 
 build:
-  number: 7
-  skip: true  # [cuda_compiler_version == "11.2"]
+  number: 0
+  # Temporarily skipping builds on some failing python 3.8 versions (see #76)
+  skip: true  # [cuda_compiler_version == "11.2" or (py<39 and linux and not aarch64 and not ppc64le and cuda_compiler_version != "None")]
   string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
   string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version == "None"]
   script:
+    # Make the compilers look for headers in the conda include directory
+    # so that nvcc finds crypt.h, which is now installed separatedly from python
+    # xref: https://github.com/conda-forge/linux-sysroot-feedstock/issues/52
+    - export CPATH=${BUILD_PREFIX}/include  # [linux and py<39]
```

Comment:
Can you remove this as we are currently skipping these versions? We normally should not look for includes in `BUILD_PREFIX`

Reply:
I'll do as you ask; if the issue is indeed fixed upstream now then hopefully it will pass.

Reply:
@xhochy several builds failed when I removed this section, so I have reverted the change and put it back. Are you happy with this workaround remaining, or would you prefer to skip all linux py38 builds?